packages/backend-proxy-middleware/test/base/proxy.test.ts

```diff
@@ -474,9 +474,9 @@ describe('proxy', () => {
             const backend = { url: '', path: '/my/path' } as LocalBackendConfig;
             try {
                 await generateProxyMiddlewareOptions(backend);
-            } catch (error) {
+            } catch (error: any) {
```

Comment:
This test confuses me. Will it pass if no error is thrown as expected?

Reply:
Added `expect.assertions(2)` in 111a307c9819a04795cd33565b9807f9d770cfa2

Reply:
Thanks Austin. I know this PR only added the type declaration in the catch clause but this test could have been written differently to start with. We're having to touch this test in the PR and it makes sense to clean it up.

@zdravko-georgiev, tagging you as this may have been an oversight.

Something like this works better?
```typescript
        test('throw an error if proxyOptions.target is not defined', async () => {
            const backend = { url: '', path: '/my/path' } as LocalBackendConfig;
            await expect(() => generateProxyMiddlewareOptions(backend)).rejects.toThrow(
                `Unable to determine target from configuration:\n${JSON.stringify(backend, null, 2)}`
            );
        });
```
Adding `expect.assertions(2)` works but if it's two places to change if the number of assertions change in the future - makes the test a little fragile.

Reply:
Thanks @ullasholla updated in 357015f2a22918a31c9f7e50d5568fd9fecf8411